modules/nf-core/ucsc/liftover/tests/main.nf.test

```diff
@@ -0,0 +1,58 @@
+nextflow_process {
+
+    name "Test Process UCSC_LIFTOVER"
+    script "../main.nf"
+    process "UCSC_LIFTOVER"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ucsc"
+    tag "ucsc/liftover"
+    config "./test.config"
+
+    test("genome - bed") {
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test' ], // meta map
+                    file(params.modules_testdata_base_path + "genomics/homo_sapiens/genome/genome.bed", checkIfExists: true)
+                ])
+                input[1] = Channel.of(file(params.modules_testdata_base_path + "genomics/homo_sapiens/genome/genome.chain.gz", checkIfExists: true))
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.lifted).match() },
+                { assert file(process.out.unlifted[0][1]).exists() }
```

Comment:
why not snapshotting the filename?

Reply:
unlifted file can be empty. When I match() it, nf-core lint was complaining 

`md5sum for empty file found: d41d8cd98f00b204e9800998ecf8427e `